src/main/webapp/scripts/map.js

```diff
@@ -12,6 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Custom Markers */
+const yellowIcon = {
+  url: "/images/yellow-marker.png", // url
+  scaledSize: new google.maps.Size(30, 40), // scaled size
+  origin: new google.maps.Point(0, 0), // origin
+  anchor: new google.maps.Point(0, 0) // anchor
```

Comment:
nit: there's no need to add a comment with the same parameter name here, we do so for example on the caller side when calling a function with an unclear parameter value ([ref](https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments))

Reply:
ah good spot, thanks!